certbot/certbot/tests/testdata/README

```diff
@@ -2,6 +2,13 @@ The following command has been used to generate test keys:
 
 	for x in 256 512 2048; do openssl genrsa -out rsa${k}_key.pem $k; done
 
+and for the elliptic curves, this command was used:
```

Comment:
I'm not really sure if this is needed.

Reply:
What do you mean here?

Reply:
I'm not really doing anything with these keys, so I'm slightly confused about this part. 

I think it should `for x in "prime256v1" "secp384r1" "secp521r1"`, and then it's fine?